src/py/flwr/cli/run/run.py

```diff
@@ -165,6 +165,7 @@ def on_channel_state_change(channel_connectivity: str) -> None:
     req = StartRunRequest(
         fab_file=Path(fab_path).read_bytes(),
         override_config=parse_config_args(config_overrides, separator=","),
+        options_config=flatten_dict(federation.get("options")),
```

Comment:
Do we want to have any kind of validation? Perhaps check if only `str` values are set?

Reply:
@danieljanes This is the case inside the `flatten_dict` function